llama_stack/apis/datasetio/datasetio.py

```diff
@@ -17,7 +17,7 @@ class PaginatedRowsResult(BaseModel):
     # the rows obey the DatasetSchema for the given dataset
     rows: List[Dict[str, Any]]
     total_count: int
-    next_page_token: Optional[str] = None
+    next_page_token: Optional[int] = None
```

Comment:
Uses PAGE_SIZE=1, `next_page_token` refers to the next row in dataset.

Reply:
I think the reason we keep this cursor as an abstract "str" is because you can then include more complex information about this cursor (think GraphQL pagination). So we should probably keep this string but then really use it as an `int` in the implementation

Reply:
ah I see, changed it back to `str`